frida-gum/src/lib.rs

```diff
@@ -70,6 +70,11 @@ pub use memory_range::*;
 mod range_details;
 pub use range_details::*;
 
+#[cfg_attr(backtrace, feature(backtrace))]
+mod backtracer;
+#[cfg_attr(backtrace, feature(backtrace))]
+pub use backtracer::*;
```

Comment:
`cfg_attr` is for documentation tags, not for enabling/disabling features. Instead use the following:
```
#[cfg(feature = "backtrace")]
#[cfg_attr(doc_cfg, doc(cfg(feature = "backtrace")))]
```

Reply:
Done.